tool/bindings_generator/dart_main.dart

```diff
@@ -20,6 +20,7 @@ void main(List<String> args) async {
   await _generateAndWriteBindings(args[0]);
 }
 
+// TODO: delete any file that hasn't been generated!
```

Comment:
Maybe we should change this to say something like `After we break apart monolithic interfaces, assert we do not generate empty files`